my-app/src/app/main-layout/content-projection/course/course.component.html

```diff
@@ -0,0 +1,24 @@
+<div class="container">
+  <div class="row">
+    <!-- Task 2  {{object.property}}-->
+    <div class="col-lg">
+    <!-- Task 2  Details-->
+    <strong>Video Course {{course.id}}. {{course.title}}</strong>
+    </div>
+    <div class="col-lg">
+      <span><i class="far fa-clock"></i> {{ course.duration }} mins</span>
+      <span style="margin-left: 20px;"><i class="far fa-calendar-alt"></i> {{ getDate(course.creationDate) }}</span>
```

Comment:
you could use datePipe in the template:
course?.creationDate | date: 'dd MMM, yyyy'

Reply:
Sure, will remove the function call and pipe it directly in template.